docs/README.md

```diff
@@ -1,106 +1,109 @@
 
 ## Getting Started
 
 ### Prerequisites
 
-- Linux server (Intel Xeon processor recommended)
-- Network Interface Card (NIC) compatible with DPDK
-- Docker installed
+- Linux server (Intel Xeon processor recommended, ex. Sapphire Rapids).
+- Network Interface Card (NIC) compatible with DPDK (ex. Intel Ethernet Controller E810-C).
+- Docker engine (recommended with Buildx toolkit) configured and installed.
+- (recommended) Update NIC drivers and firmware, more info and latest drivers [Support](#support)
 
 ### Basic Installation
 
 1. **Install Dependencies**, choose between options `a)` or `b)`.
 
-    a) The `setup_build_env.sh` script located in the `scripts` folder can be used for all-in-one environment preparation. The script is designed for Debian and was tested under `Ubuntu 20.04`, `Ubuntu 22.04` and `Ubuntu 24.04` environments.
-   To use this option just run the script after cloning the repository by typing `./scripts/setup_build_env.sh`
+    a) The `setup_build_env.sh` script located in the `scripts` folder can be used for all-in-one environment preparation. The script is designed for Debian and was tested under `Ubuntu 20.04` and `Ubuntu 22.04`, kernel version 5.15 environments.
+
+    To use this option, follow the steps `2.` and `3.`, and then run the following command:
+
+    ```bash
+    ./scripts/setup_build_env.sh
+    ```
 
     b) The following method is universal and should work for most Linux OS distributions.
 
+    - XDP-tools with eBpf: Follow the simple guide [XDP-tools](https://github.com/xdp-project/xdp-tools.git) for installation.
+    - libfabric: Follow the [libfabric from source](https://github.com/ofiwg/libfabric?tab=readme-ov-file#building-and-installing-libfabric-from-source) for installation.
     - MTL: Follow the [MTL setup guide](https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/doc/build.md) for installation.
     - gRPC: Refer to the [gRPC documentation](https://grpc.io/docs/languages/cpp/quickstart/) for installation instructions.
     - Install required packages:
+
         - Ubuntu/Debian
         ```bash
         sudo apt-get update
-        sudo apt-get install libbsd-dev
+        sudo apt-get install libbsd-dev cmake make
         ```
         - Centos stream
         ```bash
-        sudo yum install -y libbsd-devel
+        sudo yum install -y libbsd-devel cmake make
         ```
+
     - Install the irdma driver and libfabric
+
     ```bash
     ./scripts/setup_rdma_env.sh install
     ```
+
     - Reboot
 
 2. **Clone the repository**
-   ```sh
+
+   ```bash
    $ git clone https://github.com/OpenVisualCloud/Media-Communications-Mesh.git
    ```
 
 3. **Navigate to the Media-Communications-Mesh directory**
-    ```sh
+
+    ```bash
     $ cd Media-Communications-Mesh
     ```
 
 4. **Build the Media Proxy binary (run on Host)**
-    ```sh
+    ```bash
     $ ./build.sh
     ```
 
 5. **Build the Media Proxy Docker image (run in Container)**
```

Comment:
where is this container? can you please give me the address, city? ;)

Reply:
> where is this container? can you please give me the address, city? ;)

It's the XXI century, man. The container is imaginable :) It's virtual ;)